api/workAllocation2/util.ts

```diff
@@ -206,8 +206,12 @@ export function applySearchFilter(person: Person, domain: string, searchTerm: an
 
 export async function handlePost(path: string, payload: any, req: EnhancedRequest): Promise<any> {
   const headers = setHeaders(req);
-  const response: AxiosResponse = await http.post(path, payload, { headers });
+  const response: AxiosResponse = await http.post(path, payload, {headers});
   // Return the whole response, not just the data, so we can
   // see what the status of the response is.
   return response;
 }
+
+export function hasCaseAllocatorRole(authorisations: string[]): boolean {
+  return authorisations.some(authorisation => authorisation === 'case-allocator-role');
```

Comment:
Think 'case-allocator-role' may already be a constant within the application so we should use that

Reply:
checked and it's a constant in app.constants.ts

Reply:
and also in api/user/constants.ts